mapbox/mapbox-unity-sdk
# Updated to new ARKit (Fixes compilation issues for non IOS devices)

**Related issue**

Fixes #1459 

**Description of changes**

- Updated ARKit to the latest version
- Updated to the latest 2017 unity version

**QA checklists**

- [N/A] Add relevant code comments. Every API class and method should have `<summary>` description as well as description of parameters.
- [N/A] **Add tests for new/changed/updated classes and methods!!!**
- [N/A] Check out conventions in [CONTRIBUTING.md](https://github.com/mapbox/mapbox-unity-sdk/blob/develop/CONTRIBUTING.md).
- [N/A] Check out conventions in [CODING-STYLE.md](https://github.com/mapbox/mapbox-unity-sdk/blob/develop/CODING-STYLE.md)
- [TODO] Update the [changelog](https://github.com/mapbox/mapbox-unity-sdk/blob/develop/documentation/docs/05-changelog.md)
- [TODO] Update documentation.

**Reviewers**

**Todo**

There's some items that have to be done before this should be merged:

1. Review (Main question I have is, if this should be completely compiler excluded for non IOS devices):
https://github.com/devedse/mapbox-unity-sdk/blob/develop/sdkproject/Assets/UnityARInterface/Scripts/ARKitInterface.cs
2. Check if there's any need for additional documentation / dependencies
